### DISCUSSION TITLE
2023-09-15

### DISCUSSION THREAD

**Discussion by jaehee329** [woowacourse-teams/2023-haru-study#514] [daily_meeting]
### 오늘의 할 일
백엔드: 오전 강의, 오후 배포 예약
인덱스 걸기 - 관리자 페이지용 통계성 쿼리 중 유의미한 인덱스를 걸 곳을 찾는다.

프론트엔드: 배포 가능하도록 리팩터링

### 개인 일정
노아 근로 5시
룩소, 마코 시간 미정
### 내일의 할 일

**Comment by woosung1223** [comment]
### 체크인
8점 - 금요일이네요 굿

### 체크아웃
9점 - SSE가 되서 좋네요...

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 즐거운 금요일이네요~ 오늘 할 일만 잘 마무리하면 10점으로 체크아웃할 수 있을지도?

### 체크아웃
10점 - SSE 되서 기분 좋네요 ㅎㅎ

**Comment by nlom0218** [comment]
### 체크인

7점 - 테코톡이 끝나 시원 섭섭합니담 🥲

### 체크아웃

9점 - 주말이 되니 마음이 뒤숭숭하네욤

**Comment by woo-jk** [comment]
### 체크인

10점 - 벌써 금요일 ㄷㄷ

### 체크아웃
9점 - 머리아픔

**Comment by jaehee329** [comment]
### 체크인
8점 - 스레드 풀 & 커넥션 풀 어떻게 튜닝할지 보고왔어요
9점 - 주말이네요

**Comment by yeopto** [comment]
### 체크인

7점 - 지하철 이슈는 언제까지인가.

### 체크아웃

8점 - 주말이네용

**Comment by aak2075** [comment]
### 체크인

9점 - 조금 힘드네요

### 체크아웃

10점 - SSE가 돼서 좋아요
